frontend/src/components/common/RadioGroup.vue

```diff
@@ -24,7 +24,6 @@ const emit = defineEmits(['change']);
 const selectedValue = ref<string>('');
 onBeforeMount(() => {
     selectedValue.value = props.initialValue;
-    valueChanged();
```

Comment:
remove this call, otherwise it triggers the filterEnv method to call everytime we redirect the page, which will cleanup the local storage, not need to have this call here